contracts/SSVNetwork.sol

```diff
@@ -524,17 +524,6 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
 
         cluster.balance -= amount;
 
-        if (
-            cluster.isLiquidatable(
-                burnRate,
-                network.networkFee,
-                minimumBlocksBeforeLiquidation,
-                minimumLiquidationCollateral
-            )
-        ) {
-            revert InsufficientBalance();
-        }
-
```

Comment:
This is very important to keep,
If you remove this then people can withdraw past there collateral zone

The one to remove is cluster.validateClusterIsNotLiquidated();
We dont care if you are liquidated already but we do care if you put yourself to a liquidatable place

Reply:
and add a test for it